### ISSUE TITLE
Secton 2.5 seems redundant

### ISSUE THREAD

**Issue by chaals** [eea-oasis/baseline-standard#118]
Section 2.5 seems to say the same thing about a CCSM as the glossary, adding that they are important for a BPI.

That seems to be unnecessary repetition.

**Comment by Therecanbeonlyone1969** [comment]
@chaals ... not everyone reads the glossary. some redundancy is ok.

**Comment by chaals** [comment]
I disagree, but this is an editorial decision. I won't continue the discussion.
